person.rb

```diff
@@ -48,6 +50,10 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals.push(rental)
+  end
+
```

Comment:
- [ ] please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) on how to create this method. it should take the book and date as arguments and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
Why it should take a person and a date as arguments?